src/replace_letters.py

```diff
@@ -0,0 +1,34 @@
+def replace_letters(word):
+    word = word.casefold()
+    dictionary = {
+        "a": "z",
+        "b": "e",
+        "c": "e",
+        "d": "e",
+        "e": "d",
+        "f": "i",
+        "g": "i",
+        "h": "i",
+        "i": "h",
+        "j": "o",
+        "k": "o",
+        "l": "o",
+        "m": "o",
+        "n": "o",
+        "o": "n",
+        "p": "u",
+        "q": "u",
+        "r": "u",
+        "s": "u",
+        "t": "u",
+        "u": "t",
+        "v": "a",
+        "w": "a",
+        "x": "a",
+        "y": "a",
+        "z": "a",
+    }
```

Comment:
Yes and no, you were supposed to look over the alphabet to find which consonant or vowel belongs in the dict and put it there.
start with this and programmatically create the dict:
```python
char_map = {}
consonants = ["b", "c", "d", ....]
vowels = {"a", "e", "i", ....}
for char in consonants:
    # put the keys and values into the dictionary programmatically here.
```


not `dictionary isn't a good name for the variable. Variables should be named after what they contain, not the type they are.